src/easymigration/config.py

```diff
@@ -0,0 +1,46 @@
+import os
+from os.path import exists
+import yaml
+from pkgutil import get_data
+from logging import config as logconfig
+import logging
+
+configuration_file = '.easy-migration-tools.yml'
+example_configuration_file = 'example-easy-migration-tools.yml'
+configuration_file_locations = [configuration_file, os.path.expanduser('~/' + configuration_file)]
+
+
+def ensure_configuration_file_exists():
+    config = find_config_file()
+    if config is None:
+        print("No %s found; instantiating in current directory" % configuration_file)
```

Comment:
why `print` instead of `logging`?

Reply:
Copied from dans-datastation-tools. Logging is initialized by the content of this file. But `stderr` may be better than `stdout`.